client/my-sites/earn/ads/wrapper.jsx

```diff
@@ -229,39 +235,29 @@ class AdsWrapper extends Component {
 	}
 
 	renderjetpackUpsell() {
-		if ( shouldShowOfferResetFlow() ) {
-			return null;
-		}
-
 		const { siteSlug, translate } = this.props;
-		const bannerURL = `/checkout/${ siteSlug }/premium`;
+		const bannerURL = `/checkout/${ siteSlug }/${ PLAN_JETPACK_SECURITY_DAILY }`;
 		return (
 			<UpsellNudge
 				callToAction={ translate( 'Upgrade' ) }
-				plan={ PLAN_JETPACK_PREMIUM }
-				title={ translate( 'Upgrade to the Premium plan and start earning' ) }
+				plan={ PLAN_JETPACK_SECURITY_DAILY }
+				title={ translate( 'Upgrade and start earning' ) }
 				description={ translate(
-					"By upgrading to the Premium plan, you'll be able to monetize your site through the Jetpack Ads program."
+					'Make money each time someone visits your site by displaying ads on all your posts and pages.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Make money each time someone visits your site by displaying advertisements on all your posts and pages.' )` *ES Score: 8*